src/index.d.ts

```diff
@@ -9,12 +10,15 @@ declare module "express-json-validator-middleware" {
 		[K in OptionKey]?: T;
 	};
 
-	export type ValidateFunction =
-		| Function
+	type AllowedSchema =
 		| JSONSchema4
 		| JSONSchema6
 		| JSONSchema7;
 
+	export type ValidateFunction =
+		| (req: Request) => AllowedSchema
```

Comment:
TypeScript is giving me the following error for this line:

```
Function type notation must be parenthesized when used in a union type.ts(1385)
```

This fixes it:

```suggestion
		| ((req: Request) => AllowedSchema)
```

Reply:
Done 👍